Cython/Compiler/ModuleNode.py

```diff
@@ -3314,6 +3316,7 @@ def generate_type_import_call(self, type, code, import_generator, error_code=Non
             error_code = code.error_goto(error_pos)
 
         module = import_generator.imported_module(module_name, error_code)
+        code.putln("#if !CYTHON_COMPILING_IN_LIMITED_API")
         code.put('%s = __Pyx_ImportType(%s, %s,' % (
```

Comment:
I think type sharing across modules still makes sense with the limited API, don't you think? It even seems cleaner now that they are heap types.

Reply:
Yup, you are right! I just brought this back!